.travis.yml

```diff
@@ -23,12 +23,17 @@ matrix:
     - python: 2.7
       env:
         - TOXENV=jasmine
+      before_install:
+        - wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux64.tar.gz
+        - mkdir geckodriver
+        - tar -xzf geckodriver-v0.11.1-linux64.tar.gz -C geckodriver
+        - export PATH=$PATH:$PWD/geckodriver
```

Comment:
The new jasmine version requires geckodriver for Firefox (actually, it's Selenium that requires Jasmine to require it, afaik). This is currently how jasmine-py is handling it so I just dropped the code here.